tests/ensure_provider_tests.py

```diff
@@ -77,6 +77,9 @@
     "playbooks/tests_wireless.yml": {
         EXTRA_RUN_CONDITION: "ansible_distribution_major_version == '7'",
     },
+    "playbooks/tests_team.yml": {
```

Comment:
Please sort this (put this before wireless, ensure the comment is kept. Also, please add a comment about why Fedora cannot be used (please use the comment field for this ("comment": ...) and add an issue that tracks this.

Reply:
Team interface is not supported on Fedora

Reply:
create new issue #294

Reply:
Thanks!